client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -1010,26 +1010,6 @@ export const ApplicationsTable: React.FC = () => {
                                   },
                                 ]
                               : []),
-                            ...(applicationWriteAccess
-                              ? [
-                                  {
-                                    isAriaDisabled:
-                                      application.migrationWave !== null,
-                                    tooltipProps: {
-                                      content:
-                                        application.migrationWave !== null
-                                          ? t(
-                                              "message.cannotDeleteApplicationsAssignedToMigrationWave"
-                                            )
-                                          : "",
-                                    },
```

Comment:
@sjd78 did you delete lines 1013-1025 in purpose?
If the reason is to backport also #1803, please modify `applicationDropdownItems` as well

Reply:
Looks like #1803 should not be backported yet, fix version is MTA 7.1.0
https://issues.redhat.com/browse/MTA-2225

Reply:
Removed this erroneous change in the latest push.

Reply:
Oh wait -- I see the nuance now.  I'll keep the pre-1803 version of the delete button but move it to the newer location, and add the danger style